src/registrar/tests/test_models.py

```diff
@@ -458,70 +421,15 @@ def custom_is_active(self):
         with patch.object(Domain, "is_active", custom_is_active):
             # Now, when you call is_active on Domain, it will return True
             with self.assertRaises(TransitionNotAllowed):
-                application.reject()
-
-    def test_transition_not_allowed_started_ineligible(self):
-        """Create an application with status started and call reject
-        against transition rules"""
-
-        application = completed_application(status=DomainApplication.ApplicationStatus.STARTED)
-
-        with self.assertRaises(TransitionNotAllowed):
-            application.reject_with_prejudice()
-
-    def test_transition_not_allowed_submitted_ineligible(self):
-        """Create an application with status submitted and call reject
-        against transition rules"""
-
-        application = completed_application(status=DomainApplication.ApplicationStatus.SUBMITTED)
-
-        with self.assertRaises(TransitionNotAllowed):
-            application.reject_with_prejudice()
-
-    def test_transition_not_allowed_action_needed_ineligible(self):
-        """Create an application with status action needed and call reject
-        against transition rules"""
-
-        application = completed_application(status=DomainApplication.ApplicationStatus.ACTION_NEEDED)
-
-        with self.assertRaises(TransitionNotAllowed):
-            application.reject_with_prejudice()
-
-    def test_transition_not_allowed_withdrawn_ineligible(self):
-        """Create an application with status withdrawn and call reject
-        against transition rules"""
-
-        application = completed_application(status=DomainApplication.ApplicationStatus.WITHDRAWN)
-
-        with self.assertRaises(TransitionNotAllowed):
-            application.reject_with_prejudice()
-
-    def test_transition_not_allowed_rejected_ineligible(self):
-        """Create an application with status rejected and call reject
-        against transition rules"""
-
-        application = completed_application(status=DomainApplication.ApplicationStatus.REJECTED)
-
-        with self.assertRaises(TransitionNotAllowed):
-            application.reject_with_prejudice()
-
-    def test_transition_not_allowed_ineligible_ineligible(self):
-        """Create an application with status ineligible and call reject
-        against transition rules"""
-
-        application = completed_application(status=DomainApplication.ApplicationStatus.INELIGIBLE)
-
-        with self.assertRaises(TransitionNotAllowed):
-            application.reject_with_prejudice()
+                self.approved_application.reject()
```

Comment:
Q: Is there a test case that still tests `reject_with_prejudice()`? Can't find it here, though I may be missing it